idemp-reports-test/src/org/bandahealth/idempiere/report/test/ProductsAndPricesTest.java

```diff
@@ -0,0 +1,110 @@
+package org.bandahealth.idempiere.report.test;
```

Comment:
Good job on the reports but like you said, I don't see an implementer writing this logic unless we find a way of simplifying it further and possibly done on the UI.

Reply:
Thanks! I do think it will be easy to do the setup of the data for the reports for an implementer (and maybe reading the text from a PDF file) thanks to Chuck's framework, but parsing Excel would be more than I'd be willing to ask anyone.